sheepdog/transactions/upload/__init__.py

```diff
@@ -102,17 +102,20 @@ def handle_single_transaction(role, program, project, **tx_kwargs):
     """
     doc = flask.request.get_data()
     content_type = flask.request.headers.get('Content-Type', '').lower()
+    is_gdc = flask.current_app.config.get('IS_GDC')
     if content_type == 'text/csv':
         doc_format = 'csv'
-        data, errors = utils.transforms.CSVToJSONConverter().convert(doc)
+        data, errors = utils.transforms.CSVToJSONConverter(is_gdc).convert(doc)
     elif content_type in ['text/tab-separated-values', 'text/tsv']:
         doc_format = 'tsv'
-        data, errors = utils.transforms.TSVToJSONConverter().convert(doc)
+        data, errors = utils.transforms.TSVToJSONConverter(is_gdc).convert(doc)
     else:
         doc_format = 'json'
         data = utils.parse.parse_request_json()
         errors = None
-    # TODO: use errors value?
+    # TODO: use errors value somehow instead of just logging it?
+    flask.current_app.logger.error('Data conversion errors: {}'.format(errors))
```

Comment:
Isn't this logging errors even though errors could be None?

Reply:
Typical Codacity crapp, when you update code with existing PEP errors and it just flames you for it. But yeah, I will add a check for this.